std_msgs/Float64 seq_bitrate
std_msgs/Float64 mes_bitrate
std_msgs/Int32 ID
std_msgs/String link_name
std_msgs/String device_id
std_msgs/Bool active
std_msgs/Int32 mode
